Nodejs/Product/TestAdapter/JavaScriptTestDiscoverer.cs

```diff
@@ -203,10 +206,6 @@ private void DiscoverTests(Dictionary<string, HashSet<TestFileEntry>> testItems,
                 }
                 logger.SendMessage(TestMessageLevel.Informational, string.Format(CultureInfo.CurrentCulture, "Processing finished for framework '{0}'.", testFx));
             }
-            if (testCount == 0)
```

Comment:
The  test host already prints a warning when no tests are found. So no need to duplicate.